quickstart/daml/licensing/daml/Licensing/License.daml

```diff
@@ -43,6 +51,19 @@ template LicenseRenewalRequest with
   where 
     signatory user, provider
 
+    choice LicenseRenewalRequest_Cancellation : LicenseRenewalCancellation
+      with  
+        actor : Party -- Party that initiated the renewal request.
+        reasons  : Metadata
+      controller 
+        actor
+      do
+        now <- getTime
```

Comment:
Check that the actor is a stakeholder. Otherwise, you are allowing anybody to cancel the request.